docs/data/toolpad/core/components/persistent-state/LocalStorageStateZod.js

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { useLocalStorageState } from '@toolpad/core';
+import Stack from '@mui/material/Stack';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import { z } from 'zod';
+
+const schema = z.enum(['foo', 'bar', 'baz']).default('foo');
```

Comment:
The `zod` example part is not inside the preview in this one (I guess it's missing `// preview-start` actually)